service/http.proxy/src/test/java/org/kaazing/gateway/service/http/proxy/HttpProxyStreamingIT.java

```diff
@@ -52,8 +57,13 @@
         }
     };
 
+    TestRule trace = new MethodExecutionTrace();
+    TestRule timeoutRule = new DisableOnDebug(Timeout.builder().withTimeout(10, SECONDS)
+            .withLookingForStuckThread(true).build());
```

Comment:
Not related this PR, I have only seen name of the stuck thread. But we typically want stack trace. I have seen somewhere that describes on how to print stack traces.